pkg/apis/core/v1beta1/defaults_seed_test.go

```diff
@@ -25,131 +25,136 @@ import (
+			DependencyWatchdog: &SeedSettingDependencyWatchdog{
+				Weeder: &SeedSettingDependencyWatchdogWeeder{Enabled: dwdWeederEnabled},
+				Prober: &SeedSettingDependencyWatchdogProber{Enabled: dwdProberEnabled},
+			},
+			TopologyAwareRouting: &SeedSettingTopologyAwareRouting{
+				Enabled: topologyAwareRouting,
+			},
+			ExcessCapacityReservation: &excessCapacityReservation,
+			Scheduling:                &SeedSettingScheduling{Visible: scheduling},
+			VerticalPodAutoscaler:     &SeedSettingVerticalPodAutoscaler{Enabled: vpaEnabled},
+		}
+
+		SetObjectDefaults_Seed(obj)
+
+		Expect(obj.Spec.Settings.DependencyWatchdog.Weeder.Enabled).To(Equal(dwdWeederEnabled))
+		Expect(obj.Spec.Settings.DependencyWatchdog.Prober.Enabled).To(Equal(dwdProberEnabled))
+		Expect(obj.Spec.Settings.ExcessCapacityReservation).To(PointTo(Equal(excessCapacityReservation)))
+		Expect(obj.Spec.Settings.Scheduling.Visible).To(Equal(scheduling))
+		Expect(obj.Spec.Settings.VerticalPodAutoscaler.Enabled).To(Equal(vpaEnabled))
+		Expect(obj.Spec.Settings.TopologyAwareRouting.Enabled).To(Equal(topologyAwareRouting))
+	})
 
+	Describe("SeedNetworks defaulting", func() {
 		It("should default ipFamilies setting to IPv4 single-stack", func() {
 			SetObjectDefaults_Seed(obj)
 
 			Expect(obj.Spec.Networks.IPFamilies).To(ConsistOf(IPFamilyIPv4))
 		})
-	})
 
-	Describe("#SetDefaults_SeedSettingDependencyWatchdog", func() {
-		var obj *SeedSettingDependencyWatchdog
+		It("should not overwrite ipFamilies setting", func() {
+			obj.Spec.Networks.IPFamilies = []IPFamily{IPFamilyIPv6}
+			SetObjectDefaults_Seed(obj)
 
-		BeforeEach(func() {
-			obj = &SeedSettingDependencyWatchdog{}
+			Expect(obj.Spec.Networks.IPFamilies).To(ConsistOf(IPFamilyIPv6))
 		})
+	})
 
+	Describe("SeedSettingDependencyWatchdog defaulting", func() {
 		It("should default the settings", func() {
-			SetDefaults_SeedSettingDependencyWatchdog(obj)
+			SetObjectDefaults_Seed(obj)
 
-			Expect(obj.Weeder.Enabled).To(BeTrue())
-			Expect(obj.Prober.Enabled).To(BeTrue())
+			Expect(obj.Spec.Settings.DependencyWatchdog.Weeder.Enabled).To(BeTrue())
+			Expect(obj.Spec.Settings.DependencyWatchdog.Prober.Enabled).To(BeTrue())
 		})
 
-		It("should not default the seed settings because they were provided", func() {
+		It("should not default the SeedSettingDependencyWatchdog field if it is already set", func() {
 			var (
 				dwdWeederEnabled = false
 				dwdProberEnabled = false
 			)
 
-			obj = &SeedSettingDependencyWatchdog{
-				Weeder: &SeedSettingDependencyWatchdogWeeder{Enabled: dwdWeederEnabled},
-				Prober: &SeedSettingDependencyWatchdogProber{Enabled: dwdProberEnabled},
+			obj = &Seed{
+				Spec: SeedSpec{
+					Settings: &SeedSettings{
+						DependencyWatchdog: &SeedSettingDependencyWatchdog{
+							Weeder: &SeedSettingDependencyWatchdogWeeder{Enabled: dwdWeederEnabled},
+							Prober: &SeedSettingDependencyWatchdogProber{Enabled: dwdProberEnabled},
+						},
+					},
+				},
 			}
```

Comment:
For consistency reason with the above tests:
```suggestion
			obj.Spec.Settings = &SeedSettings{
				DependencyWatchdog: &SeedSettingDependencyWatchdog{
					Weeder: &SeedSettingDependencyWatchdogWeeder{Enabled: dwdWeederEnabled},
					Prober: &SeedSettingDependencyWatchdogProber{Enabled: dwdProberEnabled},
				},
			}
```